SPARKBOX-SETUP.md

```diff
@@ -0,0 +1,99 @@
+# Sparkbox - Craft
+
+A fork of Craft CMS - https://github.com/craftcms/craft
+
+## Table of Contents
+
+* [Install and Configure Craft](#install-and-configure-craft)
+    * [Install and Configure Apache](#install-and-configure-apache)
+    * [Installing MySQL](#installing-mysql)
+    * [Install Sequel Pro](#install-sequel-pro)
+    * [Sparkbox Craft Install](#sparkbox-craft-install)
+    * [Create Craft Project Database](#create-craft-project-database)
+    * [Run Craft](#run-craft)
+* [Common Issues](#common-issues)
+    * [Fields Not Saving](#fields-not-saving)
+        
+## Install and Configure Craft
+
+### Install and Configure Apache
+
+Follow the [Apache at Sparkbox](https://github.com/sparkbox/standard/tree/master/software/apache) guide in the Standard to install and configure Apache on your machine.
+
+### Installing MySQL
+
+We will not be using homebrew for mysql installations.
+
+Install the MySQL Binary: [https://dev.mysql.com/downloads/mysql/](https://dev.mysql.com/downloads/mysql/) 
+
+
+Download and follow setup instructions.
+
+**For MySQL 5.7 +**
+
+**After installing, you’ll need to do the following:**
```

Comment:
This should go in the laptop script if needed. The laptop script is intended to be rerunnable too–FYI.